exercises/concept/temperature/.docs/instructions.md

```diff
@@ -0,0 +1,26 @@
+# Instructions
+
+You are part of a firefighter organization made of international volunteers.
+Due to the international nature of the team, valuable time has been wasted over the last few calls translating between Fahrenheit and Celcius while estimating the temperature inside the burning buildings.
+To save time, and people, you decide to create a program to automatically convert between Fahrenheit and Celsius for your new teammate.
+
+## 1. Convert Farenheit to Celsius
+
+Write a function `tempToC` to convert an `Integer` temperature in Fahrenheit to its equivalent `Float` in Celsius rounded to the nearest hundredth. 
```

Comment:
Tiny nit: the function stub is there, so it's more a "fill in the blanks" that writing the function.

```suggestion
Implement the `tempToC` function to convert an `Integer` temperature in Fahrenheit to its equivalent `Float` in Celsius rounded to the nearest hundredth. 
```